### PR TITLE
Fixes the "Prisoner Management Console" circuitboard

### PR DESCRIPTION
**Pull request by BlueHNT**
## About The Pull Request
It fixes the thing mentioned in #2477 and another I noticed while working on it.

This fixes: #2477 

## Why It's Good For The Game
It works

## Testing Photographs and Procedure

<details>
<summary>Screenshots&Videos</summary>

No :)

</details>

## Changelog
:cl:
fix: fixed Prisoner Management Console
fix: fixed Gulag teleporter build values
/:cl:

### CODE REVIEW

**Comment by Bokkiewokkie** [comment]
Please format it exactly like "fixes #2477" in the description to make GitHub automatically link the issue to the PR
(there are a handful of other ways but you can look that up on git docs if you want to write it differently)

File: code/game/objects/items/circuitboards/computer_circuitboards.dm [hunk 1]
```diff
@@ -391,15 +391,15 @@
 
 //Security
 
-/obj/item/circuitboard/computer/gulag_teleporter_console
+/obj/item/circuitboard/computer/gulag_teleporter_console //NSV13 modified - fix
 	name = "labor camp teleporter console (Computer Board)"
 	icon_state = "security"
-	build_path = /obj/machinery/computer/prisoner/gulag_teleporter_computer
+	build_path = /obj/item/circuitboard/computer/gulag_teleporter_console
```
Comment (Bokkiewokkie) [inline code/game/objects/items/circuitboards/computer_circuitboards.dm:397]: This builds... itself?

**Review by Bokkiewokkie — CHANGES_REQUESTED** [review]
Since these are all just fixes you should consider porting them upstream

File: code/game/objects/items/circuitboards/computer_circuitboards.dm (hunk 1 above)
Comment (BlueHNT) [reply code/game/objects/items/circuitboards/computer_circuitboards.dm:397]: Yeah honestly I have no clue what the fuck happened there.

Comment (BlueHNT) [reply code/game/objects/items/circuitboards/computer_circuitboards.dm:397]: > This builds... itself?

There's also the one for the machine part... but I couldn't find it.

**Review by Bokkiewokkie — APPROVED** [review]

**Pull request merged**